File: GoEnv/go/common/EdocDistCert/Test/BlabEaddrGetUserIndividualClientTest.go
package main

import (
	blabClient "../../../common/EdocDistCert/Client"
	testClient "./TestConfig"
	"log"
)

// 공인전자주소 소유자정보 조회 테스트(개인)
func main() {
	log.SetFlags(log.LstdFlags | log.Lshortfile)
	err := testClient.BlabTestClientInit()
	if err != nil {
		return
	}

	clientConfig := testClient.ClientConfig
	appRes, err := blabClient.GetEaddrUser(clientConfig.Server.BaseUrl, clientConfig.Individual.Eaddr)
	if err != nil {
		log.Printf("err=%+v\n", err)
		return
	}

	log.Printf("appRes=%+v", appRes)
	log.Printf("appRes.Data=%+v", appRes.Data)
}

File: GoEnv/go/common/EdocDistCert/Test/node/BlabEaddrGetUserIndividualClientTest.mjs
import { v4 } from 'uuid';
import dateFormat, {masks} from 'dateformat';
import http from 'http';
import blabClient from './BlabClient.mjs';
import clientConfig from './BlabClientConfig.mjs';

// 개인 공인전자주소 소유자정보 조회
function BlabEaddrGetUserIndividualTest() {
    blabClient.GetEaddrUser(clientConfig.server.baseUrl,
        clientConfig.individual.eaddr)
        .then(res => {
            console.log(res)
        });
}

BlabEaddrGetUserIndividualTest();